CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ...
 
+## [1.3.1] - 2019-01-15
+### Added
+* Added missing "RangeAnnotation" to demo script by [@lsces]
+* Added "PropertyAnnotation" implementation by [@jcheron]
+
+### Changed
+* slash added before php global functions (To prevent php from searching for the function in the current namespace.)
```

Comment:
Please replace with:

```
* Added slash added before PHP global functions to improve namespace resolution speed by [@jcheron]
```

---

1. the usage of capital letters and dots was a bit chaotic
2. change author wasn't mentioned